tensorboard/components/tensor_widget/tensor-widget-impl.ts

```diff
@@ -146,12 +178,388 @@ export class TensorWidgetImpl implements TensorWidget {
     this.infoSubsection.appendChild(shapeTagDiv);
   }
 
+  /**
+   * Fill in the content of the value divs given the current slicing spec.
+   */
+  private async renderValues() {
+    if (this.valueSection == null) {
+      this.valueSection = document.createElement('div');
+      this.rootElement.appendChild(this.valueSection);
+
+      this.valueSection.addEventListener('wheel', async (event) => {
```

Comment:
Some mice have horizontal scroll wheels too.  Can you add support for a 'left-right' scroll wheel event using .deltaX?